payments/direct-debit/other-features.md

```diff
@@ -177,7 +177,7 @@ and their explanation is given below.
     "operations": [
         {
             "method": "PATCH",
-            "href": "https://{{ page.apiUrl }}/psp/directdebit/payments/3648fa94-7fd8-4e32-a14b-08d608f884ff",
+            "href": "https://{{ page.apiUrl }}/psp/directdebit/payments/{{ page.transactionId }}",
```

Comment:
Perhaps we should rename the current `apiUrl` to `apiHost` and introduce a new `apiUrl` that includes `https://`?

```suggestion
            "href": "{{ page.apiUrl }}/psp/directdebit/payments/{{ page.transactionId }}",
```